ops/model.py

```diff
@@ -1453,13 +1465,20 @@ class Relation:
     This is accessed using, for example, ``Relation.data[unit]['foo']``.
     """
 
+    active: bool
+    """Indicates whether this relation is active.
+
+    This will be ``False`` if the current event is a ``relation-broken`` event
+    assoeciated with this relation."""
```

Comment:
Typo fix, and consistency nit:
```suggestion
    associated with this relation.
    """
```